.travis.yml

```diff
@@ -11,7 +11,6 @@ env:
    - PACKAGE=vhd-format DISTRO=debian-stable OCAML_VERSION=4.04.2
    - PACKAGE=vhd-format-lwt DISTRO=ubuntu-16.04 OCAML_VERSION=4.03.0
    - PACKAGE=vhd-format-lwt DISTRO=debian-stable OCAML_VERSION=4.04.2
-   - PACKAGE=vhd-format-lwt DISTRO=centos-7 OCAML_VERSION=4.05.0
    - PACKAGE=vhd-format-lwt DISTRO=alpine OCAML_VERSION=4.06.0
 matrix:
   fast_finish: true
```

Comment:
I think we should remove `allow_failures` because we want it to work with ocaml 4.06, because that's the version we're currently using